Sources/CatbirdApp/Stores/FileResponseStore.swift

```diff
@@ -51,6 +58,31 @@ final class FileResponseStore: ResponseStore {
         return fileManger.fileExists(atPath: path, isDirectory: &isDirectory) && !isDirectory.boolValue
     }
 
+    private func fileIsSymlink(atPath path: String) -> Bool {
+        guard let fileAttributes = try? fileManger.attributesOfItem(atPath: path) else {
+            return false
+        }
+        guard fileAttributes[.type] as? FileAttributeType == .typeSymbolicLink else {
+            return false
+        }
+        return true
+    }
+
+    private func destinationOfSymlink(atPath path: String) -> String {
+        guard let symLinkDestination = try? fileManger.destinationOfSymbolicLink(atPath: path) else {
+            return path
+        }
+        if fileExists(atPath: symLinkDestination) {
+            return symLinkDestination
+        }
+        let basePath = (path as NSString).deletingLastPathComponent as NSString
+        let symlinkFullPath = basePath.appendingPathComponent(symLinkDestination)
```

Comment:
Could you please explain why do we need these lines of code?

Reply:
It is used when symlink contains relative link to another file. So we reconstruct full path from current symlink path + relative path from symlink